hrdf/a-2085-06-harallel-gripper.hrdf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I noticed a typo in the file name here -- `harallel -> parallel`

Reply:
Thanks, fixed.